### PR TITLE
[FIX] website_crm_partner_assign: show all leads for the assigned partner

### PR DESCRIPTION
**Pull request by Ephoniz**
When using the action_view_opportunity for assigned partners we should be able to see the lost leads when filtering by 'Lost', but if we don't set the active_test to False for the context we won't be able to see them in this view. So since we retrieve the leads with `self.env['crm.lead'].with_context(**action_context_origin)` we need to make sure the active_test is False to get all the leads related to the partner.

## Steps to reproduce:

1. Install website_crm_partner_assign.
2. Create at least 2 leads (one active and one lost) for a new partner.
3. Make sure we have set this partner as assigned partner.
4. Now on the partner form go to the leads action to see the leads related to the partner.

opw-3903070

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
looks good to me

the stat button should usually match what is seen when clicking on it, and the count already has active_test=False:

https://github.com/odoo/odoo/blob/e37c393d415d686584487f9ad92f062279504cf8/addons/website_crm_partner_assign/models/res_partner.py#L67

**Comment by reth-odoo** [comment]
Hello :wave: 

I assume this probably revives the issue with 1745052a5690c70dc24d40dddb2879559f5d22a0 ?

Can we instead merge the domains with `AND` as is done in that commit. This way we just display inactive records without propagating `active_test` everywhere

**Comment by Ephoniz** [comment]
Hii @reth-odoo ,

Thanks for the review! I tried to do that in the first place, but correct me If I'm wrong, since we create a temporary Lead recordset we won't be able to achieve the solution with just the domain. I have made a small change which I think should do the work and avoid the `active_test` propagation, let me know If I'm wrong. 😊

File: addons/website_crm_partner_assign/models/res_partner.py [hunk 1]
```diff
@@ -81,9 +81,11 @@ def action_view_opportunity(self):
         if not action_domain_origin:
             action['domain'] = action_domain_assign
             return action
+
+        action_context_all = dict(action_context_origin, active_test=False)
         # perform searches independently as having OR with those leaves seems to
         # be counter productive
-        Lead = self.env['crm.lead'].with_context(**action_context_origin)
+        Lead = self.env['crm.lead'].with_context(**action_context_all)
```
Comment (reth-odoo) [inline addons/website_crm_partner_assign/models/res_partner.py:88]: You could just inline `, active_test=False`

**Review by reth-odoo — APPROVED** [review]
Oh I see, yes you're right I didn't think that one through ^^'

This seems fine to me, just a suggestion to reduce it to one line :+1:

**Comment by Ephoniz** [comment]
@reth-odoo is this one ok to close? 🙂

**Review by reth-odoo — APPROVED** [review]

**Comment by nle-odoo** [comment]
robodoo delegate+

**Comment by Ephoniz** [comment]
robodoo r+

**Pull request closed without merging**